source/index.html

```diff
@@ -6,109 +6,102 @@
   </head>
   <body class="page__body">
     <header class="header">
-      <a class="header__logo-link">
-        <img src="img/logo-mobile.svg" width="33" height="37" alt="Cat energy">
-        <img class="logo-title" src="img/logo-title.svg" width="101" height="18" alt="Cat energy title">
+      <a href="#" class="header__logo-link">
+        <img class="logo-icon" src="img/logo-mobile.svg" width="33" height="37"
+          alt="Cat energy" aria-hidden="true">
+        <img class="logo-text" src="img/logo-title.svg" width="101" height="18" alt="Cat energy">
       </a>
       <nav class="main-nav">
-        <button class="main-nav__toggle main-nav__toggle--active" type="button">
+        <button class="main-nav__toggle" type="button">
           <span class="visually-hidden">Открыть меню</span>
         </button>
 
         <div class="main-nav__wrapper">
           <ul class="main-nav__list">
             <li class="main-nav__item main-nav__item--current">
-              <a href="index.html">Главная</a>
+              <a class="main-nav__link" href="index.html">Главная</a>
             </li>
             <li class="main-nav__item">
-              <a href="catalog.html">Каталог продукции</a>
+              <a class="main-nav__link" href="catalog.html">Каталог продукции</a>
             </li>
             <li class="main-nav__item">
-              <a href="form.html">Подбор программы</a>
+              <a class="main-nav__link" href="form.html">Подбор программы</a>
             </li>
           </ul>
         </div>
       </nav>
     </header>
 
     <main class="main-container">
-      <div class="hero">
-        <h1 class="main-title">Функциональное питание для котов</h1>
+      <section class="hero">
+        <h1 class="hero_title main-title">Функциональное питание для котов</h1>
```

Comment:
Кажется, элемент посеял один символ в нейминге.
Кстати, чисто технические недочеты по неймингу можно проверять [в инструменте от Йоксель](https://yoksel.github.io/html-tree/)

```suggestion
        <h1 class="hero__title main-title">Функциональное питание для котов</h1>
```